src/javascript/Create/CreateForm/create.action.spec.js

```diff
@@ -29,8 +29,11 @@ describe('create action', () => {
                 formik: {
                     submitForm: jest.fn(() => Promise.resolve()),
                     resetForm: jest.fn(),
-                    setFieldValue: jest.fn()
-                }
+                    setFieldValue: jest.fn(),
+                    setTouched: jest.fn(),
```

Comment:
as you have removed setTouched in "create.action.js", why you have added to props?

Reply:
Because it's used in validateForm function.